src/olympia/addons/tests/test_models.py

```diff
@@ -160,6 +161,9 @@ def test_clean_slug_always_slugify(self):
         b.clean_slug()
         assert b.slug.startswith('some-spaces-and'), b.slug
 
+    @patch.object(addons_models, 'MAX_SLUG_INCREMENT', 99)
```

Comment:
Given it's not an object we're tinkering with, why `patch.object` rather than just `patch`?